packages/members-api/lib/repositories/product.js

```diff
@@ -240,6 +241,7 @@ class ProductRepository {
      * @param {string} data.id
      * @param {string} data.name
      * @param {string} data.description
+     * @param {string} data.type
```

Comment:
This is also not part of the function sig 😝

Reply:
Updated